weave/trait/actuator/on_off_trait.proto

```diff
@@ -58,4 +58,14 @@ message OnOffTrait {
     };
     bool is_on = 1;
   }
+
+  // ----------- EVENTS ----------- //
+  message OnOffChangeEvent {
+    option (wdl.message_type) = EVENT;
+    option (wdl.event) = {
+      id: 1,
+      event_importance: EVENT_IMPORTANCE_PRODUCTION_CRITICAL
```

Comment:
CRITICAL events denote events that directly affect the safety/security of a a user. I don't believe this rises to that level, and as such, should be deemed 'STANDARD'.

Reply:
Right. Changed to STANDARD